packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -324,6 +331,11 @@ export class SummarizerNode implements ISummarizerNode {
                 // No base summary to reference
                 throw error;
             }
+            this.logger.logException({
+                eventName: "SummarizingWithBasePlusOps",
+                category: "error",
+            },
+            error);
```

Comment:
One thing to consider: this telemetry is great, but at the moment it does not help us understand the scope of problem because at-mentions will trip here for every file. It would be nice to have one of the following (better both):
1. A notion of "summary instance ID" such that I can count number of events per summary, as a post-processing.
   - i.e. can we add a property similar to container instance ID to logger and pass it down when we start summarization attempt?
2. Maybe do this counting in here as well
   - in general I prefer shipping code not to do post-processing that should be done on the consuming side, but if it's easy to do here, maybe we should add it

Reply:
we should file a follow up issue for this

Reply:
#4018

Reply:
#4018